ui/app/components/app/transaction-list/transaction-list.component.js

```diff
@@ -22,10 +22,24 @@ export default class TransactionList extends PureComponent {
     selectedToken: PropTypes.object,
     updateNetworkNonce: PropTypes.func,
     assetImages: PropTypes.object,
+    fetchBasicGasAndTimeEstimates: PropTypes.func,
+    fetchGasEstimates: PropTypes.func,
+    blockTime: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+    ]),
+    currentTimeEstimate: PropTypes.string,
   }
 
   componentDidMount () {
     this.props.updateNetworkNonce()
+    const promise = this.props.fetchBasicGasAndTimeEstimates()
+      .then(basicEstimates => basicEstimates.blockTime)
+
+    promise
```

Comment:
Nit: There's really no reason here to assign the promise to a local variable. A `.then` chain would be equivalent to this. The step of extracting `blockTime` isn't necessary to do async either.